.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: node_js
+node_js:
+  - "8"
+cache: yarn
+script: yarn build
+after_success:
+- test $TRAVIS_PULL_REQUEST == "false" && test $TRAVIS_BRANCH == "webpage" && bash
+  deploy.sh
+env:
+  global:
+  - secure: Fo8SzDuBGepCJH5m9lUMyuIo68j1d/KD5FGdUsMwol1a5VzgtNiuA66rINoKN9Z+5RVhkuEkub8SKTM1LDBOoO955h0xvpPvfWZQwlL0jklsHMNhiGsnGuU76uHNJXR10PeEncXw5I713rjo6Cq5zt0YlqmvfLrZQb9teJAYtWyg07vdpGgZ+xAuZZmMGjrx7tdp1c2Yys4sIZ+44ORDX/nF8Ro/1xhWdNP/yg8oKMwS2DySXhgrXNmpoN9zgISOqfZegsWJp3ivKY+0Ji8eMvXuOGTy9x32ZanYy9ssYhkZ1HsbN0iW0c32QoiUWYCMbDgVDEQMspw5/vx7Fc7V8HUmE5HzuDcWCpzorF6f2WwboWXin01uU97bKQuA9TANCUhF2+aToug4d8gFEdSrJ54PoREnRy96kPjNSjekYpEkE0pbQaTL5NTLgYTjhgs16cJk01tBgEpQjyZWotluw1ex/dBCH5+enPPynaoHbkFANCHU0r2AVbfAdvthYMXbENcJ4BRlj2G8SX035+12C5+xXL/hc8kgDUgJFEQiZLK1aa8AsMH2lMt5t/3TZeDZKCG8F9SGnNoJJ83Yn/wsT7gr46P3KhS/kqNfmmi/P7kVfcyjWmms91KLDC995sNj15Z/SCobuMdYAEf5b9yDyH/lqak8x0o2KJbA7phpl7s=
+  - secure: Oux0NLAoHR882564k+5MeRnjACjWUdDxIFw79+f/rz2iTpwbBZNNoYHui5iX9h3IUsVRbRgcrnsnHz6vFHBIF5+OUjeiy5cgQQ8OTt1GUUrawLMDIUaTecrMTdpRLyI/BZVW33VVgZaRZbft5K/SaTxiY8/d4GlKlnRWI8NSiNUt3/NauzUR8OiPApLpev9OICjQqShjzovSb1VSXd4PM7TljSWwrBTy46vufydEUJixC+F+ZOHUU+pgT0kBCZ887ObRFAUfG4Bv05jUXE3dQpX1M04z11w9qbfR+4jBM6iM54+o2ADl7DNlVOpL8Pqylm1wss6y0Q0yfnwu4vRSOaBiVV51tgjVXnVi2yOykYRxe/5QneWbRxeU2qyOlLU6GyY0yBytcWDskb7SM1/QS0jj0GrSfGGa5flc9Zummsq2AO3Rd4AA2sIAK620z/yas2l++AB0cBRCVYEzr+YXo0z4p4+a8bJupfDQEgUOwjU37b/hK10WLLdtVfk0/dbigTNJxO1l93rcNlgz0pzBCbfarhfKzFEKnPLEcBKj3OBWlLQKZC3Tk9m8Jx0wcqXmkQclpWcJx6D1jWxDH/XKGOJvTUxV6sw/Nhs7kcXGjfcLCcPxGXms17MuDxxEau9HEtW/Hb+ynMU76AIGH8BS7QE0+azRKzvVd+i2z6vNCG8=
```

Comment:
Why do we need multiple `secure` entries?

Reply:
it contain three items email, name and token. Encrypting email and name doesn't make much sense though, because they are visible in github